js/components/skipControls.js

```diff
@@ -0,0 +1,276 @@
+      var clientRect = this.domElement.getBoundingClientRect();
+      this.props.onMount(clientRect);
+    }
+  },
+
+  /**
+   * Previous Video button click handler.
+   * @private
+   */
+  onPreviousVideo: function() {
+    if (typeof this.props.controller.rewindOrRequestPreviousVideo === 'function') {
+      this.props.controller.rewindOrRequestPreviousVideo();
+    }
+  },
+
+  /**
+   * Next Video button click handler.
+   * @private
+   */
+  onNextVideo: function() {
+    if (typeof this.props.controller.requestNextVideo === 'function') {
+      this.props.controller.requestNextVideo();
+    }
+  },
+
+  /**
+   * Skip Backward button click handler.
+   * @private
+   */
+  onSkipBackward: function() {
+    if (typeof this.props.a11yControls.seekBy === 'function') {
+      var skipTimes = Utils.getSkipTimes(this.props.skinConfig);
+      this.props.a11yControls.seekBy(skipTimes.backward, false, true);
+    }
+  },
+
+  /**
+   * Skip Forward button click handler.
+   * @private
+   */
+  onSkipForward: function() {
+    if (typeof this.props.a11yControls.seekBy === 'function') {
+      var skipTimes = Utils.getSkipTimes(this.props.skinConfig);
+      this.props.a11yControls.seekBy(skipTimes.forward, true, true);
+    }
+  },
+
+  /**
+   * Handles the mouseenter event. Given that the SkipControls have pointer-events
+   * set to 'none' in order to allow clicking through them, this event is only fired
+   * when the mouse is over a button. Whenever this happens we cancel the auto-hide
+   * controls timer.
+   * @private
+   */
+  onMouseEnter: function() {
+    this.props.controller.cancelTimer();
+  },
+
+  /**
+   * Gets a map which contains templates for each of the available buttons in
+   * this component.
+   * @private
+   * @return {object} An object whose keys are the ids of the skip buttons (as
+   * defined in the skin.json) and whose values are the button components that
+   * match each id.
+   */
+  getButtonTemplate: function() {
+    var buttonTemplate = {};
+    var skipTimes = Utils.getSkipTimes(this.props.skinConfig);
+
+    var skipBackwardAriaLabel = CONSTANTS.ARIA_LABELS.SKIP_BACKWARD.replace(
+      MACROS.SECONDS,
+      skipTimes.backward
+    );
+    var skipForwardAriaLabel = CONSTANTS.ARIA_LABELS.SKIP_FORWARD.replace(
+      MACROS.SECONDS,
+      skipTimes.forward
+    );
+
+    buttonTemplate[CONSTANTS.SKIP_CTRLS_KEYS.PREVIOUS_VIDEO] = (
+      <ControlButton
+        {...this.props}
+        className="oo-previous-video"
+        icon="nextVideo"
+        focusId={CONSTANTS.SKIP_CTRLS_KEYS.PREVIOUS_VIDEO}
+        ariaLabel={CONSTANTS.ARIA_LABELS.PREVIOUS_VIDEO}
+        disabled={!this.props.config.hasPreviousVideos}
+        onClick={this.onPreviousVideo}>
+      </ControlButton>
+    );
+
+    buttonTemplate[CONSTANTS.SKIP_CTRLS_KEYS.SKIP_BACKWARD] = (
+      <HoldControlButton
+        {...this.props}
+        className="oo-center-button oo-skip-backward"
+        icon="replay"
```

Comment:
Are skip_backward and skip_forward just using the replay icon as placeholders? Or is it our final icon?

Reply:
Yes sorry, forgot to mention this. The font icons haven't been updated yet, so I'm currently just using existing icons and flipping them. This is temporary.